src/Form/Type/SecurityRolesType.php

```diff
@@ -133,6 +133,11 @@ public function configureOptions(OptionsResolver $resolver): void
 
             'data_class' => null,
         ]);
+
+        // Symfony 2.8 BC
+        if ($resolver->isDefined('choices_as_values')) {
```

Comment:
OMG this is so much cleaner that all the other hacks!

Reply:
love it 😻